Changelog: DM-3560 Actility issue fix for enterprise connection creation (#2058)
SoftwareAG/c8y-docs: content/change-logs/device-management/actility-1-11-0-issue-fix-for-enterprise-connection-creation.md

---
date:

> title: Issue fix for enterprise connection creation   

Author: Updated.

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc--fVxjY7du
    label: actility-agent
ticket: DM-3560
version: 1.11.0
---

> The Actility device regitration using an enterprise connection was failing due to an issue with the enterprise connection creation step. This issue has been resolved by fixing the enterprise connection creation part.

Reviewer: ```suggestion
Actility device registrations using an enterprise connection were failing due to an issue with the enterprise connection creation. This issue has been resolved by fixing the enterprise connection creation step.
```

Author: Updated.
